app/groceries/grocery-list/grocery-list.component.ts

```diff
@@ -1,41 +1,25 @@
-import {Component, ChangeDetectionStrategy, EventEmitter, Input, Output, Pipe, PipeTransform} from "@angular/core";
-import {Grocery} from "../../shared/grocery/grocery";
-import {GroceryStore} from "../../shared/grocery/grocery-list.service";
-import {alert} from "../../utils/dialog-util";
-
+import { Component, ChangeDetectionStrategy, EventEmitter, Input, Output } from "@angular/core";
+import { Grocery, GroceryService } from "../shared";
+import { alert } from "../../shared";
+import { ItemStatusPipe } from "./item-status.pipe";
 declare var UIColor: any;
 
-@Pipe({
-  name: "itemStatus"
-})
-export class ItemStatusPipe implements PipeTransform {
-  value: Array<Grocery> = [];
-  transform(items: Array<Grocery>, deleted: boolean) {
-    if (items && items.length) {
-      this.value = items.filter((grocery: Grocery) => {
-        return grocery.deleted == deleted;
-      });
-    }
-    return this.value;
-  }
-}
-
 @Component({
-  selector: "GroceryList",
-  templateUrl: "pages/list/grocery-list.html",
-  styleUrls: ["pages/list/grocery-list.css"],
+  selector: "grocery-list",
```

Comment:
Should this technically be “gr-grocery-list” to be consistent with using “gr-main” in `app.component.ts`?

Not sure if the prefix is necessary here or not to adhere to https://angular.io/docs/ts/latest/guide/style-guide.html#!#02-07.